src/modules/dashboard/components/FeaturedChannelList.tsx

```diff
@@ -0,0 +1,27 @@
+import { useGetChannelDetails } from 'queries';
+import { FC } from 'react';
+import { ChannelDetailsProps } from '../configs';
+import FeaturedChannelListItem from './FeaturedChannelListItem';
+
+type FeaturedChannelListprops = {
+  channel: ChannelDetailsProps;
+};
+
+const FeaturedChannelList: FC<FeaturedChannelListprops> = (props) => {
```

Comment:
@abhishek-01k I don't this component is used anywhere. As far as I think it should be used in FeaturedNotificationComponents.